docs/conf.py

```diff
@@ -10,11 +10,17 @@
 # import sys
 # sys.path.insert(0, os.path.abspath('.'))
 
-
 project = "riot"
 
+copyright = "2020-Present Datadog, Inc"  # noqa
+
 extensions = [
-    "sphinx.ext.doctest",
     "sphinx.ext.autodoc",
+    "sphinx.ext.doctest",
+    "sphinx.ext.napoleon",
+    "sphinx_rtd_theme",
     "reno.sphinxext",
+    "sphinx_click",
```

Comment:
do you need to this something to `docs/requirements.txt`?

Reply:
Yes, good catch!